packages/components/src/components.d.ts

```diff
@@ -14,6 +14,24 @@ export { HeadingLevel } from "./components/post-collapsible/heading-levels";
 export { BackgroundColor } from "./components/post-tooltip/types";
 export { Placement } from "@floating-ui/dom";
 export namespace Components {
+    interface PostAccordion {
+        /**
+          * If `true`, only one `post-collapsible` can be open at a time.
+         */
+        "closeOthers": boolean;
```

Comment:
Nitpick: In my opinion, it's something that I would have thing to be the default behavior. I looked at a couple of other similar components and they have rather a `alwaysOpen` prop.

Reply:
Sure, we can update that. I'll check with the team

Reply:
I just changed the default behavior and called the prop `multiple` in https://github.com/swisspost/design-system/pull/2079/commits/326891f3a3cc3d8f6bbdf04b0eff1c35f74630d9.